### ISSUE TITLE
Export not working for Collection Level Script

### ISSUE THREAD

**Issue by ShahzaibIbrahim** [usebruno/bruno#1479]
I wrote some pre-request script in bruno for Collection level and when I try to export it there is no information present for collection level and thus I lose my pre-request script when importing the collection to other machine. 

![image](https://github.com/usebruno/bruno/assets/49493944/d48f33b3-f03d-47d0-8470-1636cb88e7eb)

**Comment by Ex0dus92** [comment]
This still seems to be an issue.

**Comment by mkrebser** [comment]
I am also having this issue.

**Comment by khirasoft** [comment]
Yes, I think this gap should be closed. I noticed that the export even picks up a separate JS file and includes it into the export but the collection-level script which loads that script is unfortunately not included ☹️

**Comment by sanjai0py** [comment]
Related to #1482
